src/ocm/components/clusterConfiguration/review/ReviewSummary.tsx

```diff
@@ -38,8 +44,24 @@ const ReviewSummary = ({ cluster }: { cluster: Cluster }) => {
         </TableSummaryExpandable>
       )}
 
-      <TableSummaryExpandable title={'Inventory'}>
-        <ReviewHostsInventory hosts={cluster.hosts} />
+      <TableSummaryExpandable title={'Host inventory'}>
+        <>
+          <ReviewHostsInventory hosts={cluster.hosts} />
+          <RenderIf condition={isClusterPlatformTypeVM(cluster)}>
```

Comment:
Please remind me, is platform integration with Nutanix also possible?
If it is and there is no extra setup needed with them then perhaps we shouldn't display anything in that case?

Reply:
I'm checking that, I was unaware of it.

Reply:
@jkilzi Confirmed that Nutanix also has extra configuration steps, which we show in the link ( https://access.redhat.com/solutions/6983944 for Nutanix)